tasks/69_unpack-elements/unpack-elements.js

```diff
@@ -0,0 +1,13 @@
+export function unpackArray(arrToUnpack) {
```

Comment:
as followup - what will happen if you get [[[[[[['a']]]]]]] ? how can you prevent this?